source/reference/limitations.txt

```diff
@@ -73,7 +70,6 @@ discuss your requirements.
 Unsupported Collection Types
 ----------------------------
 
-- Collections with collation are currently not supported.
 - Capped collections are currently not supported.
 - Time-series collections are not supported.
```

Comment:
> Clustered collections with [expireAfterSeconds](https://www.mongodb.com/docs/master/reference/method/db.createCollection/#std-label-db.createCollection.expireAfterSeconds) set are not supported.

This should still be a limitation. Do you know why this isn't showing up?

Reply:
@ldennis, I branched off `master` for this ticket to keep that one up to date.  After we merge, I'll cherry pick the changes into the `v1.0` branch that we're shipping.

The line you're asking about is on the `v1.0` branch but not on `master`.  I'm not sure why.

Reply:
Thanks. This looks good as long as we will keep them in-sync after merging. Thank you!